backend/app/routers/auth.py

```
@@ -4,16 +1,40 @@
+    HTTPException,
+    Header,
+    APIRouter,
+    BackgroundTasks,
+    status,
+    Request,
+)
 from fastapi.responses import JSONResponse
 from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
 from sqlalchemy.orm import Session
 from datetime import timedelta
 from typing import Annotated, List, Optional
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)